README.md

```diff
@@ -14,47 +14,47 @@ In order to run the FireFly CLI, you will need a few things installed on your de
 ## Install the CLI
 
 ```
-$ go install github.com/kaleido-io/firefly-cli@latest
+$ go get github.com/kaleido-io/firefly-cli/...
```

Comment:
Not entirely sure this wildcard solution works, since it's tricky to test `go get` locally..? But I think it does. Definitely works for `go install ./...` locally.

Reply:
Probably could also directly use `github.com/kaleido-io/firefly-cli/ff` instead of the wildcard.